README.md

```diff
@@ -9,13 +9,30 @@
 A processing environment for HyP3 Plugins in AWS.
 
 ## Table of contents
+- [Developer Setup](#developer-setup)
 - [Deployment](#deployment)
   - [Prerequisites](#prerequisites)
   - [Stack Parameters](#stack-parameters)
   - [Deploy with CloudFormation](#deploy-with-cloudformation)
-- [Testing the API](#testing-the-api)
 - [Running the API Locally](#running-the-api-locally)
 
+## Developer Setup
+
+1. Clone the repository
+   ```
+   git clone git@github.com:ASFHyP3/hyp3.git
+   cd hyp3
+   ```
+2. Create and activate a conda environment
+   ```
+   conda env create -f environment.yml
+   conda activate hyp3
+   ```
```

Comment:
Is `make install` no longer necessary?

Reply:
Are you asking if we should remove the `install` command from the Makefile?

Reply:
@asjohnston-asf No, but that's an interesting idea. Or maybe the `install` target should just run `conda env create -f environment.yml`. But that's probably beyond the scope of this PR.